### DISCUSSION TITLE
Maximum payload size for route parameters?

### DISCUSSION THREAD

**Discussion by chillaxdev** [react-navigation/react-navigation#11249] [Q&A]
From the documentation at https://reactnavigation.org/docs/params/

I couldn't find the maximum payload size that can be passed as a param to a Stack Navigator route.

Out of curiosity what would be the ceiling limits?

Any answer is appreciated.

**Comment by Ektilth** [comment]
I think there is no maximum size, because we tried it with 1 Million characters and it worked without problems other than the initial lag because of the massive data size.
If you are experiencing issues with pushing params to the route parameters, then it's maybe a parentheses-related bug, present in expo router too:
https://github.com/expo/expo/issues/26664
